### ISSUE TITLE
Conditional rule-based styling

### ISSUE THREAD

**Issue by KWLangendorf** [neo4j-labs/neodash#909]
**Is your feature request related to a problem? Please describe.**

I have a table display and need to indicate if a various cells in a record has changed value compared to the previous. For this I have a a set of diff columns. But I cannot make different colouring of the changed call based on the value of the diff columns. 

**Describe the solution you'd like**
I would like to define which column should have the colour green/red based on the value in the diff column.
![image](https://github.com/neo4j-labs/neodash/assets/26305820/f7cadd73-bdb1-4b6a-8c90-198fcd02db05)
![image](https://github.com/neo4j-labs/neodash/assets/26305820/c8462016-4795-4c83-9faa-1fa36bea6764)
![image](https://github.com/neo4j-labs/neodash/assets/26305820/467d4ee8-ac79-4720-81ea-0f4f12088def)

**Comment by mariusconjeaud** [comment]
Discussed with the team. The goal is to say :

`IF column1 = "something" THEN column2.Cell TextColor = green`

**Comment by KWLangendorf** [comment]
Yes. Or could be background colorMvh KirstenDen 26. jun. 2024 kl. 14.20 skrev MariusC ***@***.***>:﻿
Discussed with the team. The goal is to say :
IF column1 = "something" THEN column2.Cell TextColor = green
